swap_meet/clothing.py

```diff
@@ -1,2 +1,19 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, age=None, category="Clothing", condition=0):
```

Comment:
Actually, category should not be in the `__init__` function. The category should always be "Clothing", you don't want the user to be allowed to send in some random value!